docs/guides/development/Limits.md

```diff
@@ -10,107 +10,107 @@ order: 140
-To set this up, the account owner has two options:
+Users with an Enterprise plan have custom API request limits. Depending on the case, it can be counted per hour or per minute.
 
-1. Go to the Bell Icon at the top of your Dashboard, then click on Notifications Center and select Account Settings.
+Contact [Voucherify Sales](https://www.voucherify.io/contact-sales) to learn more about the Enterprise plan.
 
-2. Alternatively, click the pencil icon under one of the charts. This will take you directly to the API usage notification settings.
+## Notifications
 
-To get email notifications, follow these steps:
+You can receive notifications in your app and through email when you use a certain percentage of your redemptions, API calls, or bulk API calls. The account owner can configure these thresholds and notifications in the API Usage tab in the Project settings.
 
-1. Visit Notifications Center > Account Settings.
+If you are the account owner, set this up as follows:
 
-2. Click on Show details next to the notifications you want to activate.
+1. Click the bell icon in the left sidebar and go to Notifications Center to select Account Settings.
+2. Alternatively, in the Team Settings, Subscription tab, click the pencil icon under one of the charts. This will take you directly to the API usage notification settings.
 
+To get email notifications:
+1. Go to Notifications Center and Account Settings.
+2. Click Show details next to the notifications you want to activate.
 3. Turn on the switch next to Email.
-
 4. Set the percentage usage limit that will trigger the notification.
-
 5. Click Add email to specify the email address where you want to receive notifications.
-
 6. Save your settings.
 
-For in-app notifications, simply toggle the switch next to "In app."
+For in-app notifications, toggle the switch next to "In app."
 
 Notifications for webhooks usage will be sent if the usage exceeds the limit for an hour, even though they are counted in daily periods. As a result, you will be informed in advance about any suspicious user activity.
 
-> 📘 What happens if you exceed your limits?
+> 📘 What Happens If Limits Are Exceeded?
 >
-> You can't exceed the number of API calls (both hourly and monthly) defined in your pricing plan. Exceeding this limit will block your access to the API immediately.
+> You cannot exceed the number of API calls (both hourly and monthly) defined in your pricing plan. Exceeding this limit will block your access to the API immediately.
+>
+> However, the Enterprise plan can use the automatic subscription upgrade that will upgrade their subscription plan if there is a risk of exceeding the limit. This ensures smooth service operation in case of sudden higher traffic. Contact [Voucherify Sales](https://www.voucherify.io/contact-sales) to learn more about the Enterprise plan.
```

Comment:
tutaj mowimy o upgrade to the higher pricing tier, based on the pricing tiers defined in the Order Form.

Reply:
Poprawka w https://github.com/voucherifyio/voucherify-openapi/pull/840/commits/8c44a2f25a932ba49e1154ad8b7579f8427df958